### PR TITLE
Portfolio: Accessibility check

### PR DESCRIPTION
**Pull request by Emmy-github-webdev**
## Portfolio: Accessibility check.

In the accessibility check I:

- Test Page title: No issue found
- Test Image text alternative ** (<img alt="alt text"): Give distinct text to all the image text alternative
- Check Text headings: No issue found
- Check Color contrast: Update h2 for social media color to pass accessibility test
- Check Resize: No issue found
- Check Interaction: No issue found
- Check Moving content: No issue found
- Check Multimedia: No issue found
- The basic structure of the page: No issue found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,15 +23,15 @@
     <nav>
```
Comment (carlos-ssh) [inline index.html:23]: Provide a ``<title>`` with similarity to these examples:
- Emmy's portifolio
- Emmy full-stack webdeveloper

File: index.html [hunk 2]
```diff
@@ -50,10 +50,10 @@ <h2>LET’S CONNECT</h2>
...
               <div class="social-media-image">
                 <ul>
-                  <li>  <a href="#"><img src="assets/images/mobile/twitter.svg" alt="image"></a></li>
-                  <li>  <a href="#"><img src="assets/images/mobile/Linkedin-icon.svg" alt="image"></a></li>
+                  <li>  <a href="#"><img src="assets/images/mobile/twitter.svg" alt="Twitter-Icon"></a></li>
+                  <li>  <a href="#"><img src="assets/images/mobile/Linkedin-icon.svg" alt="Linkedin-Icon"></a></li>
                   <li><a href="#"><img src="assets/images/mobile/m.svg" alt="image"></a></li>
-                  <li><a href="#"><img src="assets/images/mobile/github.svg" alt="image"></a></li>
+                  <li><a href="#"><img src="assets/images/mobile/github.svg" alt="GitHub-Icon"></a></li>
                   <li></li>
```
Comment (carlos-ssh) [inline index.html:55]: - Remove empty lines.

File: index.html [hunk 3]
```diff
@@ -195,11 +195,11 @@ <h2>LET'S CONNECT</h2>
             </div>
```
Comment (carlos-ssh) [inline index.html:192]: **Kindly work with the design guidelines provided in your Figma template.**
- Provide margins and required paddings:
Screen Shot 2021-10-05 at 15 17 51

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Emmy-github-webdev , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**